kratos/includes/accessor.h

```diff
@@ -0,0 +1,248 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Alejandro Cornejo
+//                   Riccardo Rossi
+//                   Carlos Roig
+//
+
+# pragma once
+
+// External includes
+
+// Project includes
+#include "geometries/geometry.h"
+#include "includes/process_info.h"
+#include "includes/node.h"
+
+namespace Kratos
+{
+///@addtogroup KratosCore
+///@{
+
+///@name Kratos Classes
+///@{
+
+class Properties;
+
+/**
+ * @class Accessor
+ * @ingroup Kratos Core
+ * @brief This class defines the way a certain property is accessed.
+ * @author Alejandro Cornejo, Riccardo Rossi and Carlos Roig
+ */
+class KRATOS_API(KRATOS_CORE) Accessor
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    /// Node type definition
+    using NodeType = Node<3>;
+
+    /// Geometry type definition
+    using GeometryType = Geometry<NodeType>;
+
+    /// Pointer definition of NodeSearchUtility
+    KRATOS_CLASS_POINTER_DEFINITION(Accessor);
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /// Default constructor.
+    Accessor(){};
+
+    /// Destructor.
+    virtual ~Accessor() {}
+
+    ///@}
+    ///@name Operations
+    ///@{
+    
+    /**
+     * @brief Custom method to retrieve double type properties
+     * @param rVariable The variable considered (double type properties)
+     * @param rProperties The properties considered
+     * @param rGeometry The geometry considered
+     * @param rShapeFunctionVector The shape function of the GP considered
+     * @param rProcessInfo The process info considered
+     * @return The double type properties
+     */
+    virtual double GetProperty(
```

Comment:
I would call this `GetValue` to be consistent with the other containers

otherwise the access would be:
`GetProperties().GetAccessor().GetProperty()` which I believe is quite confusing

(or maybe `FastGetValue` to increase the speed :eyes: :runner: @RiccardoRossi  xD)

Reply:
I agree with that, I put that name without thinking much hehe